### DISCUSSION TITLE
Is DNF also supported for uosc/update?

### DISCUSSION THREAD

**Discussion by NaiveInvestigator** [tomasklaen/uosc#723] [General]
I saw that apt is supported but I can't find any mention whether DNF and pacman are supported too or not.

**Comment by christoph-heinrich** [comment]
Afaik as long as your mpv doesn't run in some isolated environment it should work fine on linux.

**Comment by tomasklaen** [comment]
We just detect where the config folder is and update that. So as long as it's in the standard `~/.conf/mpv`, you're fine.
